web/styles/portico/billing.css

```diff
@@ -1,5 +1,11 @@
 :root {
     --color-background-modal: hsl(0deg 0% 98%);
+    /* Exit buttons are sometimes Cancel, but sometimes
+       other "Nah, forget it" actions. */
+    --color-exit-button-text: hsl(0deg 0% 0%);
+    --color-exit-button-border: hsl(300deg 2% 11% / 30%);
+    --color-exit-button-background: hsl(0deg 0% 100%);
+    --color-exit-button-background-interactive: hsl(0deg 0% 97%);
 }
```

Comment:
Hmm, I'm not sure about this commit; previously, these were declared in `modal.css` because they're primarily intended to be used with the other modals CSS defined there; it seems safer to just leave them there, so that exactly bundles that include `modal.css` can use them.